src/common.jl

```diff
@@ -10,20 +10,20 @@ printvec(io::IO, a::AbstractVector) = printarr(io, a')
 printarrln(io::IO, a::AbstractArray) = (printarr(io, a); println(io))
 printvecln(io::IO, a::AbstractVector) = (printvec(io, a); println(io))
 
-# centralize 
+# centralize
 
-centralize(x::AbstractVector, m::AbstractVector) = (isempty(m) ? x : x - m)::typeof(x)
```

Comment:
Have you checked that the type of the result is correctly inferred? It would be good to have tests for that using `@inferred` for common input types.

Also, remove the code rather than commenting it out.